superbench/config/azure_ndv5.yaml

```diff
@@ -0,0 +1,305 @@
+# SuperBench Config
```

Comment:
It's like a separated PR for NDv5 and MI300?

Reply:
seperated into a new PR